bindata/network/multus/multus.yaml

```diff
@@ -139,6 +139,96 @@ spec:
           value: "/usr/src/multus-cni/rhel8/bin/"
         - name: DEFAULT_SOURCE_DIRECTORY
           value: "/usr/src/multus-cni/bin/"
+      containers:
+      - name: kube-multus
+        image: {{.MultusImage}}
+        command: ["/entrypoint.sh"]
+        args:
+        - "--multus-conf-file=auto"
+        - "--multus-autoconfig-dir=/host/var/run/multus/cni/net.d"
+        - "--multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig"
+{{- if eq .DefaultNetworkType "OpenShiftSDN"}}
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf"
+{{- else if eq .DefaultNetworkType "OVNKubernetes"}}
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf"
+{{- end}}
+        - "--cleanup-config-on-exit=true"
+        - "--namespace-isolation=true"
+        - "--multus-log-level=verbose"
+        - "--cni-version=0.3.1"
+        - "--additional-bin-dir=/opt/multus/bin"
+        - "--skip-multus-binary-copy=true"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 150Mi
+        securityContext:
+          privileged: true
+        terminationGracePeriodSeconds: 10
+        volumeMounts:
+        - name: system-cni-dir
+          mountPath: /host/etc/cni/net.d
+        - name: multus-cni-dir
+          mountPath: /host/var/run/multus/cni/net.d
+        - name: cnibin
+          mountPath: /host/opt/cni/bin
+        env:
+        - name: KUBERNETES_SERVICE_PORT
+          value: "{{.KUBERNETES_SERVICE_PORT}}"
+        - name: KUBERNETES_SERVICE_HOST
+          value: "{{.KUBERNETES_SERVICE_HOST}}"
+      volumes:
+        - name: system-cni-dir
+          hostPath:
+            path: {{ .SystemCNIConfDir }}
+        - name: multus-cni-dir
+          hostPath:
+            path: {{ .MultusCNIConfDir }}
+        - name: cnibin
+          hostPath:
+            path: {{ .CNIBinDir }}
+        - name: os-release
+          hostPath:
+            path: /etc/os-release
+            type: File
+        - name: cni-binary-copy
+          configMap:
+            name: cni-binary-copy-script
+            defaultMode: 0744
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus-additional-cni-plugins
+  namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the Multus networking component on each node.
```

Comment:
Can we tweak this text so folks can distinguish from the description used for `openshift-multus`?

Reply:
Thanks for the eye, updated description.